src/reducers/ComparisonSlice.ts

```diff
@@ -90,6 +90,13 @@ export const selectProcessedResults = createSelector(
   },
 );
 
+export const selectNewRevisions = createSelector(
+  (state: RootState) => state.selectedRevisions,
+  (selectedRevisions) => {
+    return selectedRevisions.new.map((item) => item.revision);
+  },
+);
```

Comment:
I'd even suggest something like this:
```suggestion
export const selectNewRevisions = createSelector(
  (state: RootState) => state.selectedRevisions.new,
  (newSelectedRevisions) => {
    return newSelectedRevisions.map((item) => item.revision);
  },
);
```

so that the selector doesn't need to be recomputed if just the "old" revision changes.

Reply:
I see that in the reducer they all change when one changes, so currently the behavior I describe above would still happen, but this can be fixed separately.